egui/src/widgets/plot/mod.rs

```diff
@@ -115,22 +121,48 @@ impl PlotMemory {
 
 // ----------------------------------------------------------------------------
 
+#[derive(Copy, Clone, PartialEq)]
+struct Cursor {
+    orientation: Orientation,
+    point: f64,
```

Comment:
I think it is worth documenting the struct as a whole, but also the unit of `point`

Reply:
Reading how this is used, I think it would be more clear as:

```rust
enum Cursor {
    Horizontal { y: f64 },
    Vertical { x: f64 },
}
```